.github/workflows/base-commit-message-checker.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - name: Check subject line length
         uses: gsactions/commit-message-checker@v2
         with:
-          pattern: '^.{1,72}(\n|$)'
+          pattern: '^((Revert )?.{1,72}(\n|$)|(Revert ").{1,72}"(\n|$))'
```

Comment:
Does

```suggestion
          pattern: '^(Revert "?)?.{1,72}(\n|$)'
```

work?

Reply:
Yeah I agree the proposed version is a bit complex, what you suggest mostly work apart from:
- If original commit really have 72 chars, would not work, since the last ` " ` would be counted as extra char
- Would allow: Revert "something    (without end quotes) 

Might still be good enough, just not entirely correct, at the cost of an easier regex. Either way is still an improvement from the original I guess so I would be fine either way.

One more option is 
```
'^(Revert "?)?.{1,72}"?(\n|$)'
```
to still have the issue of open and not closed quotes (or vice-versa) and one more risk of allowing commits like: SOMETHING"  
But this way we do not fail when reverting a 72char commit using the UI

Reply:
Well, this check is to prevent overly long subject lines. I consider checking for the balance of quotes out of scope for this particular check :) So can you simplify the regex accordingly?